Python_Engine/Query/EmbeddedPythonHome.cs

```diff
@@ -33,8 +33,7 @@ public static partial class Query
 
         public static string EmbeddedPythonHome()
         {
-            var appdata = Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData);
-            var installDir = Path.Combine(appdata, "BHoM", Compute.EMBEDDED_PYTHON);
+            var installDir = Path.Combine(@"C:\ProgramData", "BHoM", Compute.EMBEDDED_PYTHON);
```

Comment:
Would you mind if we get the folder from the environment variable set?
```suggestion
            var programData = Environment.GetFolderPath(Environment.SpecialFolder.CommonApplicationData);
            var installDir = Path.Combine(programData, "BHoM", Compute.EMBEDDED_PYTHON);
```